rules/S3584/cfamily/metadata.json

```
@@ -3,7 +3,8 @@
   "type": "BUG",
   "code": {
     "impacts": {
-      "RELIABILITY": "BLOCKER"
+      "RELIABILITY": "BLOCKER",
+      "SECURITY": "BLOCKER"
```

> Dynamically allocated memory should be released

```suggestion
      "RELIABILITY": "BLOCKER",
      "SECURITY": "MEDIUM"
```

Could be used for DoS.